docs/terraformoptions.adoc

```diff
@@ -250,6 +250,51 @@ EOT
 |
 |oke-vcn
 
+|===
+== Reuse VCN
+
+[stripes=odd,cols="1m,4d,2m,2m", options=header,width="100%"]
+|===
+|Parameter
+|Description
+|Values
+|Default
+
+|create_vcn
+|Whether to create the VCN.
+|true/false
+|true
+
+|vcn_display_name
```

Comment:
display_name is not unique. We shouldn't use this.

Reply:
This option is provided for usecase in devops where they dont have to update for different environment.

Reply:
display name is kept for now .